arches/app/views/search.py

```diff
@@ -349,6 +350,7 @@ def get_dsl_from_search_string(request):
     return JSONResponse(dsl)
 
 
+@csrf_exempt
```

Comment:
what's the need for this?

Reply:
POST requests can fail for users who aren't authenticated.